frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -307,6 +302,27 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
     makeOCSRequest();
   };
 
+  const handleStorageClass = (sc: StorageClassResourceKind) => {
+    setStorageClass(sc);
+    const provisioner: string = sc?.provisioner;
+
+    if (provisioner === NO_PROVISIONER) {
+      setOsdSize('1'); // for baremetal environment, set requested capacity as 1 Byte
+
+      const [pvsData, pvsLoaded, loadError] = pvResult;
+
+      if (pvsLoaded && pvsData) {
+        const pvs = getSCAvailablePVs(pvsData, getName(sc));
+        const availableCapacity = humanizeBinaryBytes(calcPVsCapacity(pvs)).string;
+        setPVsAvailableCapacity(availableCapacity);
+      } else if (loadError || pvsData.length) {
+        setPVsAvailableCapacity(pvsCapacityState.NOT_AVAILABLE);
+      }
+    } else {
+      setOsdSize('2Ti');
```

Comment:
Can you create  an enum of osd sizes in `osd-size-dropdown.tsx` and then use those values. I see we need these values often now.

something like this:
```
const SUPPORTED_OSD = {
  'Small' ='512Gi',
  'Medium'='2Ti',
  'Large' = '4Ti',
  'Baremetal' = '1'
}
```

Reply:
There is already an enum `OSD_CAPACITY_SIZES` for these supported sizes. Make sense to create jsut for default osd sizes i.e. 2Ti and 1

Reply:
Okay but we can modify that too, lets keep all supported configuration at one place.
We can use these enums in that too and that way we  only need to change at one place, when changes are required.

Reply:
The structure is different and I don't think its a good idea to mix for two different things. I have already created `  defaultRequestSize` enum for handling the default request size.